lib/components/RunFrame.tsx

```diff
@@ -232,13 +232,15 @@ export const RunFrame = (props: Props) => {
         .then(() => ({ success: true }))
         .catch((e: any) => {
           // removing the prefix "Eval compiled js error for "./main.tsx":"
-          const message = e.message.split(":")[1]
+          const message: string = e.message.includes(":")
+            ? e.message.split(":")[1]
```

Comment:
What if the error message has multiple colons?

```suggestion
            ? e.message.replace(/[^:]+:/, "")
```